src/bin/calaos_server/IO/xPL/xPLLib/Address.cpp

```diff
@@ -0,0 +1,200 @@
+/*** LICENCE ***************************************************************************************/
```

Comment:
xPLLib seems to be an external Library, it would make sense to put it in lib directory; We did it already for libquickmail or uvw.

Reply:
Actually the wago code does the same, the modbus lib is in IO/Wago

Reply:
What do you prefers ? If the lib is use only by the IO, it's better to put it in a sub folder of the IO ?

Reply:
I would say keep it like it is, it's ok for me. So the xpl code is at one and only place

Reply:
yep ok for me too